x-pack/plugins/security_solution/public/timelines/components/flyout/index.tsx

```diff
@@ -37,6 +37,25 @@ const FlyoutComponent: React.FC<OwnProps> = ({ timelineId, onAppLeave }) => {
     getTimelineShowStatus(state, timelineId)
   );
 
+  const [focusOwnership, setFocusOwnership] = useState(true);
+  const searchListener = useRef(new AbortController());
+
+  const onOutsideClick = useCallback((event) => {
+    setFocusOwnership(false);
+    const classes = event.target.classList;
+    if (classes.contains('euiFieldSearch')) {
+      window.setTimeout(() => event.target.focus(), 0);
```

Comment:
Why the `setTimeout`?

Reply:
This is to ensure the outsideClickDisables handler in the focus trap runs before this